app/assets/stylesheets/fhdds.scss

```diff
@@ -37,17 +37,17 @@ input[type=password]:focus {
     box-shadow: inset 0 0 0 2px;
 }
 
-.form-control-error, 
-textarea.form-control-error, 
-input[type=text].form-control-error, 
+.form-control-error,
+textarea.form-control-error,
+input[type=text].form-control-error,
 input[type=password].form-control-error {
     border-width: 2px;
     border-color: #d4351c;
 }
 
-.form-control-error.form-control-error:focus, 
-textarea.form-control-error:focus, 
-input[type=text].form-control-error:focus, 
+.form-control-error.form-control-error:focus,
+textarea.form-control-error:focus,
+input[type=text].form-control-error:focus,
 input[type=password].form-control-error:focus {
     border-width: 2px;
     border-color: #0b0c0c;
```

Comment:
Is the style sheet needed because the components were only meant to have single questions and not conditionals & the style sheet is the only way to have the errors show correctly?